CONTRIBUTING.md

```diff
@@ -310,6 +310,18 @@ configuration files in `usr/etc/arangodb3/`, and so on.
 For building the ArangoDB Starter (`arangodb` executable), see the
 [ArangoDB Starter repository](https://github.com/arangodb-helper/arangodb).
 
+### Building Locally
+
+Recommended for Debian-based Linux. First, install dependencies:
+- `clang-19`
+- `libomp-19-dev`
+- `liblapack-dev`
+- `libopenblas-dev`
+- `cmake` (at least 3.21)
+- `libssl-dev`
```

Comment:
Also: `libstdc++-14-dev` is required

Reply:
Note: If changing only MD files there is actually no need to run full CI pipeline. We should address this.